jQuery/honeyHeadquarters.js

```diff
@@ -1,4 +1,27 @@
 // $.ajax('https://sheetsu.com/apis/v1.0/64b5c3f8').done(function(data) {
-$.ajax('http://localhost:3000/citizens').done(function(data) {
-  console.log(data);
+$.ajax('http://localhost:3000/citizens').done(function(citizens) {
+  function getUniqueValuesForField(field, citizens) {
+    const values = ['All'];
```

Comment:
Ooh. `const` is a no-no. You're making assumptions about your environment here. Chrome supports `const`, and `let` and all your other fancy es6 stuff. 

best to use `var` for maximum compatibility between browsers. 
Otherwise you risk just straight up erroring in other browsers.